openedx/core/djangoapps/credit/urls.py

```diff
@@ -0,0 +1,26 @@
+"""
+URLs for the credit app.
+"""
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from .views import create_credit_request, credit_provider_callback
+
+urlpatterns = []
+
+if settings.FEATURES.get("ENABLE_CREDIT_API"):
```

Comment:
Move this to `lms/urls.py` so that future developers don't have to dig too deep into the code to find it.